pandas/compat/_optional.py

```diff
@@ -15,42 +15,42 @@
 # Update install.rst & setup.cfg when updating versions!
 
 VERSIONS = {
-    "bs4": "4.11.1",
-    "blosc": "1.21.0",
-    "bottleneck": "1.3.4",
+    "bs4": "4.11.2",
+    "blosc": "1.21.3",
+    "bottleneck": "1.3.6",
     "dataframe-api-compat": "0.1.7",
-    "fastparquet": "0.8.1",
-    "fsspec": "2022.05.0",
+    "fastparquet": "2022.12.0",
+    "fsspec": "2022.11.0",
     "html5lib": "1.1",
     "hypothesis": "6.46.1",
-    "gcsfs": "2022.05.0",
+    "gcsfs": "2022.11.0",
     "jinja2": "3.1.2",
-    "lxml.etree": "4.8.0",
-    "matplotlib": "3.6.1",
-    "numba": "0.55.2",
-    "numexpr": "2.8.0",
+    "lxml.etree": "4.9.2",
+    "matplotlib": "3.6.3",
+    "numba": "0.56.4",
+    "numexpr": "2.8.4",
     "odfpy": "1.4.1",
-    "openpyxl": "3.0.10",
-    "pandas_gbq": "0.17.5",
-    "psycopg2": "2.9.3",  # (dt dec pq3 ext lo64)
+    "openpyxl": "3.1.0",
+    "pandas_gbq": "0.19.0",
+    "psycopg2": "2.9.6",  # (dt dec pq3 ext lo64)
     "pymysql": "1.0.2",
-    "pyarrow": "7.0.0",
-    "pyreadstat": "1.1.5",
+    "pyarrow": "10.0.1",
```

Comment:
FWIW, the pyarrow PDEP said "When the minimum version of PyArrow is bumped, PyArrow will be bumped to the highest version that has been released for at least 2 years.". But pyarrow 10.0 was only released 1 year ago.

(and in general I also think 1 year is too short for pyarrow)